frontend/src/Signup.js

```diff
@@ -0,0 +1,68 @@
+import styles from './auth.module.css';
+import classNames from 'classnames';
+import axios from 'axios';
+import {useState} from 'react';
+
+function SignUp() {
+  const [userEmail, setUserEmail] = useState ('');
+  const [userPassword, setUserPassword] = useState ('');
+  const [userConfPassword, setUserConfPassword] = useState (''); 
+
+  const signUp = async (_event) => { // called when SIGN UP button is clicked
+    const response = await axios.post('/sign-up', {
+      email: userEmail,
+      password: userPassword,
+      confPass: userConfPassword
+    });  
+    console.log("signup was successful for ", response.data)
```

Comment:
this one is probably okay to keep in. although maybe you could play with doing this as a `console.alert` instead